### PR TITLE
Update page_06.rst

### PR DESCRIPTION
**Pull request by anujsoftdel**

### CODE REVIEW

File: source/Pages/page_06.rst [hunk 1]
```diff
@@ -230,3 +230,27 @@ Default log file size is around 34mb.
 To change the file size, “log4cpp.properties” needs to be changed. Please change the limit highlighted above. The max file size mentioned here is in bytes. Please identify number of bytes as per file size needed and set the value here. 
 
 Please run script 03 Build_Run_UWC.sh   after changing “log4cpp.properties”.
```
Comment (nagdeepgk) [inline source/Pages/page_06.rst:232]: remove this additionally space

File: source/Pages/page_06.rst [hunk 2]
```diff
@@ -230,3 +230,27 @@ Default log file size is around 34mb.
...
+
+6.7.1 Pre-processor flag to be used for enabling/disabling KPI-App on high performance/low performance processor
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+1. By default, pre-processor flag UWC_HIGH_PERFORMANCE_PROCESSOR is disabled in Kpi App for debug & release mode.
+
+2. To enable KPI-App on high performance processor in release mode, go to Sourcecodekpi-tacticKPIAppReleasesrc directory and open subdir.mk file. Add the option “-      DUWC_HIGH_PERFORMANCE_PROCESSOR” in below line where GCC compiler is invoked.
```
Comment (nagdeepgk) [inline source/Pages/page_06.rst:239]: Formatting "Sourcecodekpi-tacticKPIAppReleasesrc" with spaces

File: source/Pages/page_06.rst [hunk 3]
```diff
@@ -230,3 +230,27 @@ Default log file size is around 34mb.
...
+
+   g++ -lrt -std=c++11 -I../$(PROJECT_DIR)/include -I/usr/local/include -I../$(PROJECT_DIR)/include/yaml-cpp -O0 -g3 -Wall -c -fmessage-length=0 -MMD -MP -MF”$(@:%.o=%.d)” -  MT”$(@)” -o “$@” “$<” ..
```
Comment (nagdeepgk) [inline source/Pages/page_06.rst:241]: Can add -DUWC_HIGH_PERFORMANCE_PROCESSOR in the g++ line for reference

File: source/Pages/page_06.rst [hunk 4]
```diff
@@ -230,3 +230,27 @@ Default log file size is around 34mb.
...
+
+4. To disable pre-processor flag in Kpi App, remove the option “-DUWC_HIGH_PERFORMANCE_PROCESSOR” added in steps 2 & 3 for both Release and Debug mode.
+
+6.7.2 High and Low Performance Processors in Intel Processors Family
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In this context, Intel high performance processor family includes i3/i5/i7/i9 and low performance family includes Atom processors.
+
+Intel i processors have multiple cores for more power and smoother multi-tasking. This family is having several cores which significantly increases the speed at which certain programs run. Core i3 processors have two cores, Core i5 and Core i7 processors have four cores. High performance of processors isn’t only determined by clock speed and number of cores, other factors like cache memory size also play a part. Core i3 processors have 3-4MB, i5s processors have 6MB and core i7s processors have 8MB of cache memory
```
Comment (nagdeepgk) [inline source/Pages/page_06.rst:254]: @shivaji, @anujsoftdel : Are we sure about these cache sizes for we are mentioning for i3, i5 i7 etc. ? Also are we sure about the no: of cores that we are calling out here ?  @arunkk-intel @vkb1 @vats-in  Do you have any feedback here ?

File: source/Pages/page_06.rst [hunk 5]
```diff
@@ -230,3 +230,27 @@ Default log file size is around 34mb.
...
+
+Intel Atom is a single-or dual-core chip, which means that it has one or two processor units. Intel Atom are made for embedded systems where low power consumption is required. Intel Atom processors are more energy efficient and use less power.
```
Comment (nagdeepgk) [inline source/Pages/page_06.rst:256]: @anujsoftdel  @shivaji, Again are we sure about this single or 2 core ?

File: source/Pages/page_06.rst (hunk 4 above)
Comment (bdasamara) [reply source/Pages/page_06.rst:254]: this is just incorrect and needs to be removed.  simply say Atom processors are lower power and lower performance, Core class processors are higher power and higher performance.  Both can be found in edge devices for ruggedized environments.

File: source/Pages/page_06.rst (hunk 5 above)
Comment (bdasamara) [reply source/Pages/page_06.rst:256]: it is best not to make commentary about the Intel product family or SKU's with adjectives.  It should simply say Intel Atom processors can be used where less compute power is needed.

(for general information Intel's Atom processors are primarily 2 and 4 core.  there are a small number of product SKU's with a single core.)

File: source/Pages/page_06.rst (hunk 3 above)
Comment (bdasamara) [reply source/Pages/page_06.rst:241]: any comments about compiler flags and such have to be reviewed by solution architect.

Comment (nagdeepgk) [reply source/Pages/page_06.rst:241]: @bdasamara CC: @vats-in : These flags have already been updated & released a part of the code in UWC-1.6 release.
This is just for the documentation update in user guide it's being updated here.

File: source/Pages/page_06.rst (hunk 4 above)
Comment (nagdeepgk) [reply source/Pages/page_06.rst:254]: @bdasamara CC @vats-in Agree with you Dean. I too had the same concern here, hence had asked Shivaji to get this removed.
@shivaji, @anujsoftdel : Can you please make changes as Dean suggessted.

File: source/Pages/page_06.rst (hunk 5 above)
Comment (nagdeepgk) [reply source/Pages/page_06.rst:256]: @bdasamara  CC @vats-in Agree with Dean. 
@anujsoftdel @shivaji: PLease update as Dean suggested.

**Comment by nagdeepgk** [comment]
Closing this PR because, i have submitted this changes in another PR (created by Intel) https://github.com/open-edge-insights/uwc-docs/pull/38 & merged, by incorporating my review comments myself.

**Pull request closed without merging**